container_build.sh

```diff
@@ -37,6 +37,21 @@ rm_container_image() {
   fi
 }
 
+add_entrypoint() {
+    containerfile=$(mktemp)
+    cat > ${containerfile} <<EOF
+FROM $2
+ENTRYPOINT [ "/usr/bin/$3.sh" ]
+EOF
+    eval $1 --no-cache -t "$2-$3" -f ${containerfile} .
```

Comment:
on my end it's failing on `--no-cache` parameter not found in podman CLI

Reply:
it seems $1 $2 $3 are not the expected parameters
1 = podman
2 = build
3 = whisper-server